docs/_static/spec/rasa.yml

```
@@ -398,6 +409,7 @@ paths:
       security:
       - TokenAuth: []
       - JWT: []
+      operationId: evaluateModelStories
```

To keep it consistent with the operation id of `/model/test/intents`, I would use
```suggestion
      operationId: testModelStories
```